### ISSUE TITLE
Swap memory is memory!

### ISSUE THREAD

**Issue by lylythechosenone** [KrispyCamel4u/SysMonTask#69]
Currently, the app only shows physical memory. I only have 8GB physical memory, but I have a 16GB swap file on my SSD. I would like the app to show this too.

**Comment by KrispyCamel4u** [comment]
In Memory resource tab, the used swap space is shown in form of <used>/<max swap>. 
Is it like you are suggesting that a separate tab used be used to show the swap like the physical memory?

**Comment by lylythechosenone** [comment]
No, I would just like to have swap memory as a part of the memory graph.

**Comment by IrenvanderBeekPXL** [comment]
I think thats not a good idea. SWAP is much slower than RAM, and should not be confused with RAM
